internal/indexer/indexer.go

```diff
@@ -337,30 +372,9 @@ func (i *Indexer) positionAndDocument(p *packages.Package, pos token.Pos) (token
 	return position, d, true
 }
 
-// markRange sets an empty range identifier in the ranges map for the given position.
-// If a  range for this identifier has already been marked, this method returns false.
```

Comment:
@efritz It looks like the point of this method was to prevent multiple definitions/references associated with the same range. With the addition of symbols, we now expect most definitions' ranges to *also* be associated with a symbol, which means those ranges will be emitted in the `indexSymbols` stage. Therefore, this method is no longer valid (keeping it as is would mean we don't emit definitions whose ranges have already been emitted in the symbols stage). Deleting it didn't break any tests. Let me know if there should still be a check that enforces non-duplication for some subset of range-adjacent vertices emitted.

Reply:
I have to think a little harder about this, but will definitely help testing on some examples to make sure there's no big performance/space regression here.

Reply:
I'm unsure why you need multiple definitions to a range - is this because you're nesting tags inside of the range vertex itself?

Reply:
@beyang can you answer "I'm unsure why you need multiple definitions to a range - is this because you're nesting tags inside of the range vertex itself?"

Reply:
(I want to make sure I understand)

Reply:
answers: https://github.com/sourcegraph/lsif-go/pull/142